services/src/main/java/com/chatbot/services/AsyncService.java

```diff
@@ -0,0 +1,151 @@
+package com.chatbot.services;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.chatbot.services.protobuf.ChatServiceRequestOuterClass.ChatServiceRequest;
+import com.chatbot.services.protobuf.ChatServiceRequestOuterClass.ChatServiceRequest.ChatClient;
+import com.chatbot.services.protobuf.TriggerEventNotificationOuterClass.TriggerEventNotification;
+import com.google.cloud.dialogflow.v2.QueryResult;
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AsyncService {
```

Comment:
Add comment with class description.

For each method/function add a comment with the description